OWNERS

```diff
@@ -6,3 +6,5 @@ approvers:
  - digambar15
  - russellb
  - maelk
+ - ashughorla
```

Comment:
This is the approvers list, you need to add a separate list for reviewers. See e.g. https://github.com/metal3-io/baremetal-operator/blob/master/OWNERS for an example.

Reply:
Sure @zaneb, I will raise PR for this also.

Reply:
@zaneb, Done. Removed both ID from approvers and added to reviewers.